### DISCUSSION TITLE
PDM support

### DISCUSSION THREAD

**Discussion by 0x0fe** [pschatzmann/stm32f411-i2s#2] [Q&A] [answered]
Does your I2S library support PDM? specifically i want to try it with ICS-41350 MEMS mic. 
There is an official precompiled library, https://github.com/shrayanth-zz/STM32F4/blob/c38cda5475dc24f06106fa563596dea97006836e/STM32F4_Discovery_IO_Toggle/Utilities/STM32F4-Discovery/libPDMFilter_GCC.a but i do not know how well it could integrate, not sure if it relies on the ST HAL or if it is directly low level (i presume using SIMD instructions).

**Comment by pschatzmann** [comment, marked as answer]
No, but Feel free to contribute a generic PDM implementation e.g. to my audio-tools
